README.md

```diff
@@ -97,6 +90,15 @@ message EnableVolumeReplicationResponse {
 }
 ```
 
+#### Error Scheme
+
+| Condition | gRPC Code | Description | Recovery Behavior |
+|-------------------|-------------------|-------------------|-------------------|
+| Missing required field | INVALID_ARGUMENT | Indicates that a required field is missing from the request. | Caller MUST fix the request by adding the missing required field before retrying. |
```

Comment:
do we need to pass in error code number as well ?

Reply:
@Madhu-1 Can you pls confirm?

Reply:
good to document the error codes as well. documented here https://grpc.github.io/grpc/core/md_doc_statuscodes.html

Reply:
done